CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to passenger-docker
+
+First off, thank you for considering contributing to passenger-docker. It's people like you that make Open Source a great place to be. Now let's make sure your contribution gets merged!
+
+### Contributions we happily welcome
+We look for contributions improving documentation, bug triaging, or writing tutorials, as well as bug reports and patches.
+
+### Contributions we are not looking for
+Please, don't use the issue tracker for support questions. Instead consider Stack Overflow.
```

Comment:
Might I suggest adding a Stack Overflow tag as well, I personally really like [this README](https://github.com/thbar/kiba/blob/master/README.md) forwarding to https://stackoverflow.com/questions/tagged/kiba-etl

Reply:
would a link to the 'newest' questions and answers concerning 'passenger docker' work for you? eg https://stackoverflow.com/search?tab=newest&q=passenger%20docker @blijblijblij

Reply:
yes 👍 for sure!